geode-core/src/main/java/org/apache/geode/internal/offheap/OffHeapStoredObjectAddressStack.java

```diff
@@ -48,17 +50,24 @@ public void offer(long e) {
     synchronized (this) {
       OffHeapStoredObject.setNext(e, topAddr);
       topAddr = e;
+      length++;
     }
   }
 
+  public int getLength() {
```

Comment:
since this class represents a "stack" I think "size" would be better than "length". In Java collections like this usually use size and length is used for arrays and strings.